_sass/blocks/explore/_map.scss

```diff
@@ -62,7 +62,8 @@ eiti-slider {
   }
 
   .mask {
-    fill: $mid-blue;
+    // fill: $mid-blue;
+    fill: rgba(198,233,255, 0.85);
```

Comment:
Color literals like `rgba(198, 233, 255, 0.85)` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Commas in function arguments should be followed by a single space